transformer_engine/pytorch/csrc/userbuffers/userbuffers.cu

```diff
@@ -4,18 +4,21 @@
  * See LICENSE for license information.
  ************************************************************************/
 
+#include <stdio.h>
+#include <assert.h>
+
 #include <cuda.h>
 #include <cuda_runtime.h>
+#include <cuda_fp8.h>
+
 #if __CUDA_ARCH__ >= 800
 #include <cuda_bf16.h>
 #define half nv_bfloat16
```

Comment:
For future reference, this line is dubious since it allows `half` to pollute the other headers. It would be better to split it up so that the macro definition happens after all the header includes.